src/result/result.py

```diff
@@ -287,3 +309,43 @@ def result(self) -> Result[Any, Any]:
         Returns the original result.
         """
         return self._result
+
+
+def as_result(
+    *exceptions: Type[TBE],
+) -> Callable[[Callable[P, R]], Callable[P, Result[R, TBE]]]:
+    """
+    Make a decorator to turn a function into one that returns a ``Result``.
+
+    Regular return values are turned into ``Ok(return_value)``. Raised
+    exceptions of the specified exception type(s) are turned into ``Err(exc)``.
+    """
+    # Note: type annotations for signature-preserving decorators via ParamSpec
+    # are currently not fully supported by Mypy 0.930; see
+    # https://github.com/python/mypy/issues/8645
+    #
+    # The ‘type: ignore’ comments below are for our own linting purposes.
+    # Calling code works without errors from Mypy, but will also not be
+    # type-safe, i.e. it will behave as if it is calling a function like
+    # f(*args: Any, **kwargs: Any)
```

Comment:
So this will be the only incompleteness of mypy remaining, right?

Reply:
I'm not seeing any `type:ignore` left in this code here. Can we remove that part of the comment on line 327?

Reply:
ah, this comment was a left-over. with mypy 0.930 (very recent) none of them are needed anymore. removed.